packages/documentation/docusaurus.config.js

```diff
@@ -35,9 +35,9 @@ try {
 const customCss = [
   ...libCss,
   require.resolve('./src/css/custom.css'),
-  require.resolve('./src/css/search.css'),
-  require.resolve('./src/css/api-table.css'),
-  require.resolve('./src/css/cards.css'),
+  require.resolve('./src/css/search.scss'),
+  require.resolve('./src/css/api-table.scss'),
+  require.resolve('./src/css/cards.scss'),
```

Comment:
Why renamed to scss if there are not changes inside the styles files related to scss?

Reply:
because imho it does not make any sense to use css and scss in combination. but - your choice.

Reply:
But you didn't changed `custom.css` then